packages/api/src/cli.ts

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+
+import "./dotenv";
+import makeApp from "./index";
+import makeStreamInfoSvc from "./app/stream-info/stream-info-app";
+import parseCli from "./parse-cli";
+
+if (require.main === module) {
+  const args = parseCli();
+  if (args.streamInfoService) {
+    makeStreamInfoSvc(args);
+  } else {
+    makeApp(args);
+  }
```

Comment:
@iameli I've made some changes to this in order to get `-j` hopefully working better. Do you think this fixes the issues you described on https://github.com/livepeer/livepeer-infra/issues/890 ?

Reply:
Yep, probably!

Reply:
I'll try it out - concerned that we might get errors from strict mode `yargs` about some of the environment variables but we'll see